DTDL/siemens_industrial_edge_source.json

```diff
@@ -0,0 +1,42 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:io:tributech:source:siemens_industrial_edge;1",
+    "@type": "Interface",
+    "extends": "dtmi:io:tributech:source:base;1",
+    "displayName": "Siemens Industrial Edge Source",
+    "contents": [
+        {
+            "@type": "Property",
+            "name": "MetadataDiscoveryTopic",
+            "schema": "string",
+            "writable": false,
+            "displayName": "Metadata discovery topic",
+            "description": "The MQTT topic for discovery of metdata for tags/data points from the Siemens IE Databus (e.g. \"ie/m/j/simatic/v1/s7c1/dp\" where ie: industrial edge, m: metadata, j: json, s7c1: SIMATIC S7 Connector instance 1, dp: datapoint)."
+        },
+        {
+            "@type": "Property",
+            "name": "MaxBatchSize",
+            "schema": "integer",
+            "writable": false,
+            "displayName": "Max batch size",
+            "description": "The maximum number of values to send as batch in one message to the EdgeAgent. (e.g. 1024)"
+        },
+        {
+            "@type": "Property",
+            "name": "FlushInterval",
```

Comment:
same

Reply:
There is no `FlushInterval` on "Base stream" nor "Siemens Industrial Edge Stream".